coopAndLastFM2.js

```diff
@@ -155,6 +165,12 @@ function draw() {
 		fill(0);
 		text("Getting Data");
 	}
+	
+	if(songTitle) {
+		textFont(font);
+		text(songTitle, width/2, height/2);
+	}
+	
 }
```

Comment:
If we have songTitle data, draw the title using the font thats set on the variable